crates/bevy_internal/src/default_plugins.rs

```diff
@@ -68,7 +68,7 @@ impl PluginGroup for DefaultPlugins {
 
         #[cfg(feature = "bevy_winit")]
         {
-            group = group.add(bevy_winit::WinitPlugin::default());
+            group = group.add(bevy_winit::WinitPlugin::<bevy_winit::WakeUp>::default());
```

Comment:
I'm unsure how to achieve that: `T` requires `Event`, so it can be sent inside the loop and handled in a system. Would you happen to know how I can do that?

Reply:
Ok, I needed to set the default type in the plugin, and specify `::<WinitPlugin>` in the `group.add` call. Another one in my toolbelt, thanks for the suggestion.